### ISSUE TITLE
Show deletion date on 'Recover deleted Glosses' page in the admin

### ISSUE THREAD

**Issue by richardbank** [Signbank/NGT-signbank#149]
Recovery of deleted glosses is available in the admin through a button on the top right of the page  Home -› Dictionary -› Glosses. This list is presented and sorted by date of entry of the gloss.
- Show date of deletion in addition to date of entry
- Sort list descending

**Comment by Woseseltops** [comment]
Related to https://github.com/Signbank/NGT-signbank/issues/188

**Comment by Woseseltops** [comment]
This is about the functionality of a seperate Django extension called reversion. 

> Sort list descending

I discovered an option to flip the order of the list, so I guess that one is covered.

> Show date of deletion in addition to date of entry

Turns out this would really force me to dive deep into the reversion code and extend it (which is why this issue was not taken up by anyone for so long I guess :) ). To make sure the database would not go corrupt, I would probably start a separate django project just for this. I suspect a multiweek struggle this working; knowing this, do we really still want it @ocrasborn ? Because I secretly think it would rarely be used.
